app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java

```diff
@@ -637,4 +640,12 @@ public static boolean checkRange(final CharSequence seq, final int... indices) {
     public static boolean checkRange(final int length, final int... indices) {
         return indices != null && indices.length >= 2 && inRange(0, length, indices) && indices[1] > indices[0];
     }
+
+    // Check if keyboard open. Only available after android 11 :(
```

Comment:
I have had something around for older android versions, but it very often is/was unreliable - virtual keyboard detection is horrible, better not trust it generally

Reply:
Yeah, that's why I have it set to use a Boolean - true, false and null for indeterminate